composer.json

```diff
@@ -19,7 +19,8 @@
         "rospdf/pdf-php": "0.12.41",
         "mk-j/php_xlsxwriter": "0.35",
         "phpmailer/phpmailer": "v6.0.1",
-        "parsecsv/php-parsecsv": "0.4.5"
+        "parsecsv/php-parsecsv": "0.4.5",
+        "squizlabs/php_codesniffer": "^3.2"
```

Comment:
¿Para qué el codesniffer?

Reply:
Está explicado en la descripción:

- Added dependency to composer.json for squizlabs/php_codesniffer to work automatically on Travis-CI.

Hace falta para esto:

https://github.com/NeoRazorX/facturascripts/pull/245/files#diff-354f30a63fb0907d4ad57269548329e3R14

Así se le puede indicar que revisar y que ignorar

Reply:
¿Esa dependencia no puede ir en require-dev?

Reply:
Si, y ya está hecho el cambio desde un rato después que lo dijeras ;)